### PR TITLE
Feat/spasht 4.0

### PR DESCRIPTION
**Pull request by vivekaindia**
# Feat spasht Licenses Integrated Successfully
#### PR depends on

> #1451 

## Description

Successfully fixed spasht db schema and integrated Licenses

### Changes

- Fixed Table Issues
- Fixed Current Spasht UI
- Started Agent to fetch the response  from ClearlyDefined harvest endpoint
- Successfully separated Licenses
- Added Spasht 'Sp' to License UI

## How to test

`sudo make install`

### CODE REVIEW

File: src/spasht/agent/Makefile [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# Copyright 2019
-# Author: Vivek Kumar<vvksindia@gmail.com>
+# Copyright Siemens AG 2014
```
Comment (GMishx) [inline src/spasht/agent/Makefile]: Please update the copyright

File: src/spasht/agent/SpashtAgent.php [hunk 2]
```diff
@@ -29,11 +42,334 @@ class SpashtAgent extends Agent
...
+      $uploadAvailable = $this->searchUploadIdInSpasht($uploadId);
+
+      if($uploadAvailable == false)
+      {
+        $file = fopen('/home/fossy/abc.json','w');
+        fwrite($file,"no data available");
+        fclose($file);
+
+        return false;
+      }
```
Comment (GMishx) [inline src/spasht/agent/SpashtAgent.php]: Please remove the debugging statements.

File: src/spasht/agent/SpashtAgent.php [hunk 3]
```diff
@@ -29,11 +42,334 @@ class SpashtAgent extends Agent
...
+          else
+          {
+            if(!empty($l->getId()))
+            {
+              $this->dbManeger->insertTableRow('license_file',['agent_fk' => $agentId,'pfile_fk' => $key['pfileId'],'rf_fk'=> $l->getId()]);
+            }
+          }
+        }
+      }
+      fclose($file);
```
Comment (GMishx) [inline src/spasht/agent/SpashtAgent.php]: Please remove the debugging statements.

File: src/spasht/agent/Makefile (hunk 1 above)
Comment (vivekaindia) [reply src/spasht/agent/Makefile]: Updated in new commit of feat/spasht_5.0

File: src/spasht/agent/SpashtAgent.php (hunk 2 above)
Comment (vivekaindia) [reply src/spasht/agent/SpashtAgent.php]: Updated in Feat/spasht_5.0

File: src/spasht/agent/SpashtAgent.php (hunk 3 above)
Comment (vivekaindia) [reply src/spasht/agent/SpashtAgent.php]: Updated in Feat/spasht_5.0

**Pull request merged**